core/src/main/java/org/bitcoinj/store/SPVBlockStore.java

```diff
@@ -175,9 +179,8 @@ else if (fileLength < randomAccessFile.length())
     }
 
     private void initNewStore(Block genesisBlock) throws Exception {
-        byte[] header;
-        header = HEADER_MAGIC.getBytes(StandardCharsets.US_ASCII);
-        buffer.put(header);
+        ((Buffer) buffer).rewind();
```

Comment:
Isn't this an unrelated change that should be in a separate commit?

Reply:
Indeed. Removed.